### PR TITLE
Actualised helper/filter tutorial

### PR DESCRIPTION
**Pull request by sallyx**

### CODE REVIEW

File: pla/en/helper-loader.texy [hunk 1]
```diff
@@ -61,13 +62,14 @@ class Helpers extends Nette\Object
...
+	 * in method $template->addFilter().
 	 */
 	public function loader($helper)
 	{
-		if (method_exists($this, $helper)) {
-			return callback($this, $helper);
+		if (!method_exists($this, $helper)) {
+			return NULL;
 		}
+		return call_user_func_array(callback($this, $helper), array_slice(func_get_args(), 1));
```
Comment (fprochazka) [inline pla/en/helper-loader.texy]: this line is new to me, where have you found it?

File: pla/en/helper-loader.texy [hunk 2]
```diff
@@ -112,17 +114,21 @@ common:
...
 
 
-Than we can register `Helper loader` in BasePresenter.php by callback to our method loader() into Template::registerHelperLoader().
+Than we can register `Filter loader` in BasePresenter.php by callback to our method loader() into Template::addFilter().
 
 /--php
 abstract class BasePresenter extends Nette\Application\UI\Presenter
 {
+    /**
+    * @var \App\Helpers @inject
```
Comment (fprochazka) [inline pla/en/helper-loader.texy]: This classname might be confusing, do you think you could change that to something like `App\TemplateFilters` or `App\RuntimeFilters` ?

File: pla/en/helper-loader.texy (hunk 1 above)
Comment (sallyx) [inline pla/en/helper-loader.texy]: Somewhere in the nette forum.

**Comment by fprochazka** [comment]
Anyway, great work!

**Comment by dg** [comment]
merged 4e94552d5f983da1c8f536faae889950e5956711

**Pull request closed without merging**